tests/test_read_write/test_read_jsonl.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from clumper import Clumper
+
+
+@pytest.mark.parametrize("lines, expected", [(0, 0), (1, 1), (2, 2)])
```

Comment:
Probably good to also add a case `(None, 4)`. I'm wondering if it also makes sense to add a case `(5, 4)`. To me it feels like appropriate behavior to read in all the lines if `lines` > number of items.

Reply:
You can also do something similar for the test below.

Reply:
> Side question. I might want to add some calmcode videos on reviewing and collaboration on GitHub. Would you be opposed if I used our interactions in some examples? Feel free to say no if you don't prefer it.

No problem. I have learnt so much from your videos at calmcode so I am happy to give something back to the community, even if that means they see my mistakes.

Reply:
> Probably good to also add a case `(None, 4)`. I'm wondering if it also makes sense to add a case `(5, 4)`. To me it feels like appropriate behavior to read in all the lines if `lines` > number of items.

Regarding `(None, 4)` case: 
This is handled by existing code. 

Regarding `(5, 4)` case:
My code already handles this as iterate through each line of the file. The iteration ends at the end of the file so `lines` > number of items then the program never reaches the `break` keyword so the entire file will be read.  

Added test case to both local and cloud file.
Maybe another test case could be to read negative lines and expect an error?